raiden/console.py

```diff
@@ -214,25 +213,23 @@ def register_asset(self, token_address):
         self._raiden.register_channel_manager(channel_manager)
         return channel_manager
 
-    def ping(self, peer):
+    def ping(self, peer, timeout=0):
         """See, if a peer is discoverable and up.
```

Comment:
Because it seems that sphinx expects this "sphinx" syntax, or whatever it's called.

But by googling on what `napoleon syntax` is I came across [this](https://pypi.python.org/pypi/sphinxcontrib-napoleon/) which is a sphinx extensions which seems to allow this napoleon syntax for sphinx too.

So at this point I only can say I think it's more succint and easy to read ... but if we can use both in sphinx I don't have any strong feelings.